.github/workflows/end2end.yml

```diff
@@ -4,8 +4,11 @@
 name: End2End Testing
 on:
   push:
-    paths:
-    - 'chart/**'
+    branches: 
+      - main
+    tags:
+      - "v[012].[0-9]+.[0-9]+"
+  pull_request:
```

Comment:
Shouldn't we always run the e2e testing, on each an every push? We need to always make sure that the sparrow can be deployed.

Reply:
I've added these rules to minimize our CI minutes since we're using the Github Runners.

While we're on the topic, should we use own runners?

Reply:
Total usage of runner in the last 90 days was 33min. We have more than a 1000 each month. Let's keep the E2E tests and I'll open an issue to use a dedicated runner :)

Reply:
I agree with @puffitos. If this becomes relevant we can easily host our own runners. I also talked to to @eumel8, he also doesn't see it as an issue right now